tests/regression/test_oracle_report_with_notifier.py

```diff
@@ -0,0 +1,182 @@
+import pytest
+from brownie import Contract, accounts, chain, interface, OpStackTokenRatePusherWithSomeErrorStub, web3, reverts
+from utils.test.oracle_report_helpers import oracle_report
+from utils.config import contracts, get_deployer_account, network_name
+from utils.test.helpers import ZERO_ADDRESS, eth_balance
+from utils.evm_script import encode_error
+from typing import TypedDict, TypeVar, Any
+
+WST_ETH = "0xB82381A3fBD3FaFA77B3a7bE693342618240067b"
```

Comment:
The addresses must not be hardcoded, but taken from the config file. Otherwise the test won's pass on any network except for sepolia